### ISSUE TITLE
Support for redux-observable v0.14.1

### ISSUE THREAD

**Issue by jvbianchi** [angular-redux/redux-observable-decorator#12]
Does this library support redux-observable v0.14.1?

I'm getting:
```
warning "redux-observable-decorator@1.0.0" has incorrect peer dependency "redux-observable@ ^0.13.0".
```

**Comment by jvbianchi** [comment]
Anyone?

**Comment by e-schultz** [comment]
Hi,

I'll add support for this soon, sorry for letting this issue hang open for too long.

**Comment by e-schultz** [comment]
Hi @jvbianchi - just published v1.1.0 which updates the peer dependencies for redux-observable 0.14.0+ 

Let me know if you have any issues,

Thanks!
